src/Microsoft.Health.Fhir.Core/Microsoft.Health.Fhir.Core.csproj

```diff
@@ -79,4 +79,5 @@
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <ProjectExtensions><VisualStudio><UserProperties data_4r4_4search-parameters_1json__JsonSchema="https://beaujs.com/schema.json" /></VisualStudio></ProjectExtensions>
```

Comment:
Is this required?

Reply:
Removed. Not sure where it came from :).